static/style.css

```diff
@@ -160,6 +160,92 @@ h3 {
     color: #bbbbbb;
 }
 
+/* Story selection form styles */
```

Comment:
Adust the widths of the two horizontal panels to give the right side additional space.  Make is 2x as wide as the left side